client/src/components/Recipe.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState, useContext } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import clsx from "clsx";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
```

Comment:
You can import these material ui components using destructuring, so instead of having to have import statements per component, you can for example:
`import { Card, CardHeader, CardMedia, etc, etc } from "@material-ui/core`

Reply:
But generally, for my own components I try to write the code as you said.